runtime/doc/lua.txt

```diff
@@ -574,6 +574,15 @@ retained for the lifetime of a buffer but this is subject to change. A plugin
 should keep a reference to the parser object as long as it wants incremental
 updates.
 
+Required runtime files					*lua-treesitter-files*
```

Comment:
I worry about naming this section "lua-treesitter-files", since the files aren't just for lua.

I don't know that we always have to prefix everything with `lua` for tree-sitter stuff.

End users don't care what language we implemented it in.

Reply:
Yeah you are right, I will move that to `treesitter-files`.